partials/footer.hbs

```diff
@@ -1,3 +1,3 @@
-<footer>
+<footer class="container footer  col-xs-10 col-xs-offset-1">
```

Comment:
Here the point was to leave the page decide how to render the footer. Especially when and if we are going to add a `blogpost` template